test/compilable/deprecations.d

```diff
@@ -0,0 +1,49 @@
+/*
+REQUIRED_ARGS: -dw
+TEST_OUTPUT:
+--
+compilable\deprecations.d(37): Deprecation: struct deprecations.S is deprecated
+compilable\deprecations.d(37): Deprecation: struct deprecations.S is deprecated
+compilable\deprecations.d(39): Deprecation: struct deprecations.S is deprecated
+compilable\deprecations.d(39): Deprecation: struct deprecations.S is deprecated
+--
+*/
+
+
+deprecated struct S {}
+
+deprecated void foo()(S par)
```

Comment:
Could you also add `deprecated void foobar (T) (T par) { T inst; }` and instantiate it with a deprecated struct ?

Reply:
Done